Header header
ServoDiag[] diagnostics #array of diagnostics messages for all dynamixels on bus